src/controller.py

```diff
@@ -211,7 +208,10 @@ def run_emulator_command(self) -> dict:
             emulator.apply_settings(**settings_fields)
             return {"response": f"Applied settings on emulator {settings_fields}"}
         elif self.command == "emulator-reset-device":
-            emulator.reset_device()
+            emulator.reset_device(
+                self.request_dict.get("backup_type", messages.BackupType.Bip39),
+                self.request_dict.get("strength", None),
```

Comment:
Just a small thing here, `None` is not necessary as a second argument, `dict.get("xx")` is implicitly returning `None` when the key does not exist there